config/crd/bases/sriovnetwork.openshift.io_sriovnetworknodestates.yaml

```diff
@@ -50,6 +50,8 @@ spec:
                       type: integer
                     name:
                       type: string
+                    needVhostNet:
```

Comment:
I prefer to not changing API and always loading the tun module like how rdma_core is installed, do you see any issue loading the tun module unconditionally?

Reply:
sounds good to me :)